modules/collection/programs/obs-studio.nix

```diff
@@ -5,35 +5,31 @@
   ...
 }: let
   inherit (lib.modules) mkIf;
-  inherit (lib.options) mkOption mkEnableOption mkPackageOption;
-  inherit (lib.types) listOf package;
+  inherit (lib.options) mkEnableOption mkPackageOption;
 
   cfg = config.rum.programs.obs-studio;
 in {
   options.rum.programs.obs-studio = {
     enable = mkEnableOption "OBS Studio";
 
-    package = mkPackageOption pkgs "obs-studio" {};
+    package = mkPackageOption pkgs "obs-studio" {
+      extraDescription = ''
+        You can override the package to install plugins.
 
-    plugins = mkOption {
-      type = listOf package;
-      default = [];
-      example = [
-        pkgs.obs-studio-plugins.wlrobs #for screen capture w/wayland
-        pkgs.obs-studio-plugins.obs-vkcapture #vulkan/opengl game capture
-      ];
-      description = ''
-        A list of plugins the obs-studio package will be wrapped with.
-        Set of plugins available in nixpkgs under the obs-studio-plugins set.
+        ```nix
+        package = pkgs.wrapOBS.override {
```

Comment:
I would add a command to this, as this is not an override. But it's extremely useful nonetheless